app/Http/Middleware/HasBankAccount.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+
+use \App\Tenant as Tenant;
+
+class HasBankAccount
+{
+		/**
+		 * Handle an incoming request.
+		 *
+		 * @param  \Illuminate\Http\Request  $request
+		 * @param  \Closure  $next
+		 * @return mixed
+		 */
+		public function handle($request, Closure $next)
+		{
+			$tenant = new Tenant();
+			$hasbankaccount = $tenant->has_bank_account();      
+			if ( $hasbankaccount == FALSE){
+				return response()->json($hasbankaccount, 403);
```

Comment:
```suggestion
				use Illuminate\Validation\ValidationException;

                                throw ValidationException::withMessages(['bank account' => __('validation.custom.bank_account.not_set')]);
```

ensure you add the 'not_set' property to this [array](https://github.com/purplenimbus/edu-api/blob/master/resources/lang/en/validation.php#L132). On the back end the plan is to setup [internalization](https://laravel.com/docs/8.x/localization#replacing-parameters-in-translation-strings) so we can convert to hausa and yoruba long term